packages/core/src/payment/strategies/ppsdk/step-handler/continue-handler/continue-handler.ts

```diff
@@ -1,26 +1,31 @@
 import { FormPoster } from '@bigcommerce/form-poster';
 import { overSome } from 'lodash';
 
+import { PaymentHumanVerificationHandler } from '../../../../../spam-protection';
+import PaymentAdditionalAction from '../../../../payment-additional-action';
 import { PaymentsAPIResponse } from '../../ppsdk-payments-api-response';
 
+import { handleHumanVerification, isHumanVerification, HumanVerification } from './human-verification';
 import { handleRedirect, isRedirect, Redirect } from './redirect';
 
-export type Continue = Redirect;
+export type Continue = Redirect | HumanVerification;
 
-const isAnyContinue = overSome(isRedirect);
+const isAnyContinue = overSome([isRedirect, isHumanVerification]);
 
 export const isContinue = (body: PaymentsAPIResponse['body']): body is Continue => isAnyContinue(body);
 
 export class ContinueHandler {
     constructor(
-        private _formPoster: FormPoster
+        private _formPoster: FormPoster,
+        private _humanVerificationHandler: PaymentHumanVerificationHandler
     ) {}
 
-    handle(body: Continue): Promise<void> {
+    handle(body: Continue, callback?: (additionalAction: PaymentAdditionalAction) => Promise<void>): Promise<void> {
```

Comment:
Again, this `callback` needs to apply to all types of `continue` responses, not just `resubmit_with_human_verification`. If not, we should make it a specific callback. Otherwise, the caller might wonder why the `callback` doesn't get trigger for other continue response types, such as `redirect`.